lib/screens/loading.dart

```diff
@@ -87,8 +89,9 @@ class _LoadingScreenState extends State<LoadingScreen> {
     } else if (_showLoginScreen == false) {
       StorageSharedPrefs p = new StorageSharedPrefs();
       String far = await p.getFarStatus();
-      String email = await p.getEmail();
-      if (far == "false") {
+      String email = await p.getEmail();     
+
+      if (far == null) {
```

Comment:
Is this optimal ? I have a question here. What happens if `far` is `false` ?

Reply:
I am not getting far in response that's why its not being set to local storage